users/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Profile(models.Model):
+    adress = models.CharField(max_length = 256, blank = True)
+    phone = models.CharField(max_length = 18, default = '+')
+    avatar = models.ImageField(upload_to = 'photos/profiles', verbose_name = 'User\'s photo')
+    zip_code = models.CharField(max_length = 12)
+    user = models.OneToOneField(User, on_delete = models.CASCADE)
```

Comment:
1) Why do we need a separate table for the user's profile? I think we can extend Django's "User" model
2) Where are the migrations?